src/commands/force/mdapi/retrieve.ts

```diff
@@ -100,16 +102,16 @@ export class Retrieve extends SourceCommand {
     }),
   };
 
-  protected retrieveResult: RetrieveResult;
-  private sourceDir: string;
-  private retrieveTargetDir: string;
-  private zipFileName: string;
-  private unzip: boolean;
-  private wait: Duration;
-  private isAsync: boolean;
-  private mdapiRetrieve: MetadataApiRetrieve;
-  private flags: Interfaces.InferredFlags<typeof Retrieve.flags>;
-  private org: Org;
+  protected retrieveResult: RetrieveResult | undefined;
+  private sourceDir: string | undefined;
+  private retrieveTargetDir: string | undefined;
+  private zipFileName: string | undefined;
+  private unzip: boolean | undefined;
+  private wait: Duration | undefined;
+  private isAsync: boolean | undefined;
+  private mdapiRetrieve: MetadataApiRetrieve | undefined;
+  private flags!: Interfaces.InferredFlags<typeof Retrieve.flags>;
+  private org!: Org | undefined;
```

Comment:
do you know why this is required now? we already had strict checks enabled before
https://www.typescriptlang.org/tsconfig#strictPropertyInitialization

Reply:
update:
turns out we didn't have strict checks enabled before.